agent/wpthook/test_state.cc

```diff
@@ -349,7 +363,7 @@ bool TestState::IsDone() {
         _test_result = TEST_RESULT_TIMEOUT;

```

Comment:
don't you need to turn this off if we have a per-test timeout? otherwise we'll timeout on "whichever comes sooner"